### DISCUSSION TITLE
Important!! Looking for a Technical Lead and GitHub Management Support!

### DISCUSSION THREAD

**Discussion by drbacke** [Akkudoktor-EOS/EOS#225] [Announcements]
Hi everyone,

As many of you know, I’m leading this open-source project, which is already successfully running at my home. The system optimizes devices like inverters, batteries, and smart-home appliances, planning behavior for multiple days in advance. While the core functionality is stable and effective, there’s so much more potential to grow and reach a broader audience. To achieve this, I’m looking for someone to take over the technical leadership and GitHub management aspects of the project.

Why I’m Delegating This Role:

At the moment, I’m the bottleneck for the project’s technical development. My time is limited, and as a result, progress is slower than it could be. This isn’t fair to the community, and if we don’t address it, the project risks stagnating — which would be a real shame given its potential. By handing off the technical leadership role, we can ensure the project continues to grow and evolve, and I can focus on areas where I can bring the most value.

Responsibilities of the Technical Lead:

Repository Management: Improve code structure, clean outdated parts, and maintain consistency.

Pull Requests and Issues: Manage community feedback, review/merge pull requests, and prioritize issues.

Technical Coordination: Oversee key development tasks like GUI development, middleware integration, and modular device connections.

Roadmap Development: Collaborate with the team to create and maintain a clear roadmap for features and improvements.

My Vision for the Project:

The ultimate goal is to bring this project to a broader audience by making it more accessible and user-friendly. Here’s the vision:

1. User Interface (GUI): Develop an intuitive and visually appealing interface to replace the current Node-RED setup.

2. Middleware Integration: Enable seamless coupling with platforms like Home Assistant or similar solutions to simplify device connections and management.

3. Ready-to-Use Images: Offer pre-configured Docker images and long-term solutions with pre-installed hardware for easy deployment.

4. Device Anbindung (Integration): Standardize and expand support for various devices and protocols to ensure scalability and flexibility.

What I’ll Focus On:
I will of course continue to check and sort the pr and issues. 

If someone steps up to take on this role, I’ll be able to focus on:

Communication and Outreach: Creating content and raising awareness through my YouTube channel.

Community Engagement: Gathering feedback to steer the project in the right direction.

Strategic Planning: Preparing for the next steps, including a potential Kickstarter campaign for funding.

If you’re interested in becoming the Technical Lead or managing GitHub, please reach out!, Whether you’re an experienced developer, an active contributor, or just passionate about the project, let’s discuss how we can work together to make this vision a reality.

Let’s take this project to the next level, together — and make sure it doesn’t stall out!

**Comment by drbacke** [comment]
@Lasall  or  @b0661 Maybe one of you? 😅  I will of course continue to check and sort the pr and issues. 
It would also make sense for us to connect more directly and plan the next steps together.
But the Main Focus will be to communicate this Project and get more people ivolved (maybe also on some congresses ...)
Or maybe both of you?

**Reply by sandboxcode** [reply]
Great idea @drbacke! From next week on I can invest more time into the project and support the technical lead(ers). Let us not forget @NormannK in that regard who is also very active! 👍🏻

**Reply by NormannK** [reply]
Thanks but I’ve already declined. I believe I lack the necessary skill set, as this isn’t my area of expertise. However, I’m happy to contribute in any way I can.

**Reply by Lasall** [reply]
I would enjoy working towards the vision (in whichever role). I would feel more comfortable to share responsibilities.

**Reply by drbacke** [reply]
That sounds great. I will directly give you admin access. 
I don't think there will be much change at first. It will all happen over time, but it's good if a few people have admin rights and we share some of the organisation and management tasks 
We should organisine a meeting soon and then plan how to proceed.

**Comment by drbacke** [comment]
@michaelosthege Are you interested in joining our Core Team? I connected already @NormannK , @b0661 , @Lasall and mysself. Would be nice, if you join our discussion.

**Reply by michaelosthege** [reply]
over the holidays I can tune in and maybe help with some input regarding architecture, but beyond that I don't have the time to take more responsibility. I also don't have energy infrastructure I could apply EOS to.
